bindings/python/algorithm/expose-rnea.cpp

```diff
@@ -45,31 +45,28 @@ namespace se3
                        "Vector of external forces expressed in the local frame of each joint (size Model::njoints)"),
               "Compute the RNEA with external forces, put the result in Data and return it.",
               bp::return_value_policy<bp::return_by_value>());
-      
 
-      bp::def("nle",
+      bp::def("nonLinearEffects",
               &nonLinearEffects<double,0,JointCollectionDefaultTpl,VectorXd,VectorXd>,
               bp::args("Model","Data",
                        "Configuration q (size Model::nq)",
                        "Velocity v (size Model::nv)"),
               "Compute the Non Linear Effects (coriolis, centrifugal and gravitational effects), put the result in Data and return it.",
               bp::return_value_policy<bp::return_by_value>());
-      
-      
+
       bp::def("computeGeneralizedGravity",
               &computeGeneralizedGravity<double,0,JointCollectionDefaultTpl,VectorXd>,
               bp::args("Model","Data",
                        "Configuration q (size Model::nq)"),
-              "Computes the generalized gravity contribution g(q) of the Lagrangian dynamics.",
+              "Compute the generalized gravity contribution g(q) of the Lagrangian dynamics, put the result in data.g and return it.",
               bp::return_value_policy<bp::return_by_value>());
-      
-      
+
       bp::def("computeCoriolisMatrix",
               &computeCoriolisMatrix<double,0,JointCollectionDefaultTpl,VectorXd,VectorXd>,
               bp::args("Model","Data",
                        "Configuration q (size Model::nq)",
                        "Velocity v (size Model::nv)"),
-              "Computes the Coriolis Matrix C(q,v) of the Lagrangian dynamics.",
```

Comment:
I think computes is fine too.

Reply:
Yes, "computes" is probably better, but we use "compute [...] put" everywhere, it was for consistency. The main change was adding that the result is stored in data